.gitignore

```diff
@@ -63,3 +63,4 @@ dump.rdb
 /.project
 .generators
 .rakeTasks
+*.iml
```

Comment:
this has nothing to do with vets-json-schema

Reply:
This is ignoring files created by intellij

Reply:
+1 to getting rid of changes unrelated to the task at hand - even something as innocuous as this. It might be annoying to open a separate PR, but it helps streamline the review process.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/4594